mmv1/templates/terraform/examples/bigquery_analyticshub_subscription_basic.tf.erb

```diff
@@ -0,0 +1,38 @@
+resource "google_bigquery_analytics_hub_data_exchange" "listing" {
```

Comment:
Can you also add an update test: https://googlecloudplatform.github.io/magic-modules/develop/test/#add-an-update-test

This will verify fields update correctly.

Reply:
Done.